packages/devextreme/testing/helpers/device.js

```diff
@@ -0,0 +1,16 @@
+import devices from '__internal/core/m_devices';
+
+export const shouldSkipOnDevice = ({
+    allowedDevices = [],
+    assert = null,
+    message = null
+} = {}) => {
+    if(!Array.isArray(allowedDevices)) {
+        throw new Error('allowedDevices should be an array');
+    }
+    if(!allowedDevices.includes(devices.real().deviceType)) {
+        assert && assert.ok(true, message || 'Test should be skipped on devices not in list:  ' + allowedDevices.toString());
+        return true;
+    }
+    return false;
+};
```

Comment:
The purpose of this function is to skip or pass test according to device type.
Better to not use negatives and say everything directly.
We can use `QUnit.test.if` https://qunitjs.com/api/QUnit/test.if/
```
QUnit.test.if('render', isDesktop(), function (assert) {
  assert.strictEqual(MyApp.render(), '<p>Hello world!</p>');
});
```

Reply:
I also think it's better to use negatives because I don't want to remember all existing devices when I want to skip just on phone. I think signature gonna be "{  devices, assert, message } "

Reply:
regarding qunit.test.if - not sure, let's not do it now at least because there will be conflicts with existing wraps like testInActiveWindow